File: Python/egoing_python_and_ruby/12.Function/5.py
input_id = input("아이디를 입력해주세요. \n")
def login(id):
    members =['egoing', 'k8805','leezche']
    for member in members:
        if member == id:
            return True
    return False
if login(input_id):
    print('Hello, ' + input_id)
else:
    print('Who are you?')

File: Python/egoing_python_and_ruby/12.Function/5.rb


puts("아이디를 입력해주세요.")
input_id = gets.chomp()

def login(id)
  members =['egoing', 'k8805', 'leezche']
  for member in members do
      if member == id
        return true
        end
  end
  return false
end

if login(input_id)
  puts('Hello, '+input_id)
else
  puts('Who are you?')
end
